packages/api/test/post-build-test.js

```diff
@@ -34,14 +34,14 @@ const { deleteRepo } = require('../lib/deleter');
 const PostBuildHandler = require('../src/post-build.js');
 const client = require('../src/firestore.js');
 
-nock.disableNetConnect();
-nock.enableNetConnect(/^(?!.*github\.com).*$/); // only disable requests on github.com
-
 const assert = require('assert');
 const fetch = require('node-fetch');
 
 describe('Posting Builds', () => {
   before(async () => {
+    nock.disableNetConnect();
```

Comment:
it seems a bit weird that we  moved this logic down into the `before` statement... I find this a logic a little weird in general.

@StuartRucker why does it `enableNetConnect` for GitHub.com, but in the comment say it's disabling it? 

Ideally we shouldn't be making an external connections in tests.

Reply:
I'm not currently using nock in server-test, but the test for DELETE /repo, which does a manual redirect, was throwing an error because nock was interfering with any github calls (even though that test wasn't actually hitting the github endpoint, since it was doing a manual redirect). The test passed when run in isolation, but failed when run alongside post-build-test. Moving that logic into the before statement fixed the issue.

Reply:
```
nock.disableNetConnect();
nock.enableNetConnect(/^(?!.*github\.com).*$/); // only disable requests on github.com
```
Was the only way to allow all requests EXCEPT github.com. Basically disable everything, and then enable anything matching a pattern that is NOT github